modules/ROOT/pages/sysconfig-enabling-wifi.adoc

```diff
@@ -0,0 +1,122 @@
+= Enabling Wi-Fi
+
+The primary use for Fedora CoreOS has been driving server hardware in individual datacenters or cloud environments, which have high speed wired networking without the need for Wi-Fi enablement. Since there are many different types of wireless cards, link:https://github.com/coreos/fedora-coreos-tracker/issues/862[adding Wi-Fi enablement to Fedora CoreOS by default] would require many large firmware binaries to be installed for a non-standard use, which isn't ideal.
+
+On the other hand, Fedora CoreOS is versatile enough to run on smaller devices in IoT applications or in home labs where Wi-Fi may be required. In these cases it is easy enough to add a layer with the needed tools and firmware.
+
+== Adding Wi-Fi tools and firmware
+
+Typically enabling Wi-Fi on Fedora CoreOS involves adding the `NetworkManager-wifi` package along with the firmware package that corresponds to the wireless card in your system. Here is a list of some of the wireless firmware packages in Fedora:
+
+.Wi-Fi firmware packages in Fedora
+[source, text]
+----
+atheros-firmware - Firmware for Qualcomm Atheros WiFi/Bluetooth adapters
+b43-fwcutter - Firmware extraction tool for Broadcom wireless driver
+b43-openfwwf - Open firmware for some Broadcom 43xx series WLAN chips
+brcmfmac-firmware - Firmware for Broadcom/Cypress brcmfmac WiFi/Bluetooth adapters
+iwlegacy-firmware - Firmware for Intel(R) Wireless WiFi Link 3945(A)BG and 4965AGN adapters
+iwlwifi-dvm-firmware - DVM Firmware for Intel(R) Wireless WiFi adapters
+iwlwifi-mvm-firmware - MVM Firmware for Intel(R) Wireless WiFi adapters
+libertas-firmware - Firmware for Marvell Libertas SD/USB WiFi Network Adapters
+mt7xxx-firmware - Firmware for Mediatek 7600/7900 series WiFi/Bluetooth adapters
+nxpwireless-firmware - Firmware for NXP WiFi/Bluetooth/UWB adapters
+realtek-firmware - Firmware for Realtek WiFi/Bluetooth adapters
+tiwilink-firmware - Firmware for Texas Instruments WiFi/Bluetooth adapters
+atmel-firmware - Firmware for Atmel at76c50x wireless network chips
+zd1211-firmware - Firmware for wireless devices based on zd1211 chipset
+----
+
+For example, if a system has a Qualcomm wireless card then adding the `NetworkManager-wifi` and `atheros-firmware` packages would sufficiently enable the system for connecting to Wi-Fi.
```

Comment:
We could mention e.g. `lspci` here as a way to check which manufacturer/model the Wi-Fi adapter is.

Reply:
I would feel more comfortable with that suggestion if we shipped it in FCOS. I guess I can mention toolbx

Reply:
Ahhh yes, indeed. I had assumed it was in util-linux.